docs/sourcemaps.rst

```diff
@@ -302,6 +303,35 @@ the ``--ext`` parameter.
     *   Make sure that your minified files you have on your servers
         actually have references to your files.
 
+Using Sentry Webpack Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another way to manage releases and sourcemaps on Sentry, is to use the Sentry Webpack Plugin.
+
+* Start by creating a new authentication token under **[Account] > API**.
+* Ensure you you have ``project:write`` selected under scopes.
+* Install ``@sentry/webpack-plugin`` using ``npm``
+* Create ``.sentryclirc`` file with necessary config (see Sentry Webpack Plugin docs below)
```

Comment:
This can be confusing. @HazAT can sentry-wizard help here?

Reply:
So yeah, `sentry-wizard` could help here, it would create the file for you but it is not yet implemented for "generic" projects.
If install `@sentry/wizard` and select "Create Sentry Cli config" I could make it so it creates the file locally.
Also, I am not sure if we should promote `sentry.properties` or `.sentryclirc` or both?
@mitsuhiko @jan-auer

Reply:
I want to change sentry-cli first to always load `sentry.properties` before we promote that more.

Reply:
Thanks. Merging it for now and we'll update as we go.